### ISSUE TITLE
Default shape no more

### ISSUE THREAD

**Issue by ejiek** [dkanada/frost#264]
Since this icon pack is not planning to support adaptive shapes (#229) I see no reason to force any.

There are a lot of icons in Circle shape for no particular reason (which contradicts the `clean and simple` goal of the project).

Examples: NewPipe, MAPS.ME, Twitter, Aurora Store/Services/Droid and many more.

Sometimes we just follow the lead of an original icon (Telegram is circle shaped), but sometimes it's just out of the blue (Twitter has just a bird while in frost it's caged is a Circle).

My proposal is to avoid any "background" shape until it's a central part of design (Yandex Taxi or MuPDF viever for example). In this case it can be an acceptance criteria.

**Comment by dkanada** [comment]
I agree completely! It can actually look worse when there's almost a common shape but not quite everywhere, so for this icon pack I would prefer to avoid common shapes whenever possible.

**Comment by ejiek** [comment]
To make this issue actionable and to bring definition of when it's complete I suggest to **put "no default shape unit absolutely necessary" rule in contribution guide**.

Not sure where exactly to put it and in what form.
While we are trying to be clear about our view on the topic it's necessary not to scare contributors off =]

**Comment by dkanada** [comment]
Feel free to put it wherever :) I haven't updated that guide in ages, and there are a lot of minor changes that would improve it.
